ispyb/model/integration.py

```diff
@@ -85,3 +91,68 @@ def __str__(self):
     ('detector_distance', 'refinedDetectorDistance'),
     ('timestamp', 'recordTimeStamp'),
 ))
+
+class UnitCell():
+  '''An object representing the parameters of the unit cell I.e unit cell edges and angles
+  '''
+
+  def __init__(self, a, b, c, alpha, beta, gamma):
+    '''Unit cell object
+
+       :param a: Edge a
+       :param b: Edge b
+       :param c: Edge c
+       :param alpha: Angle alpha
+       :param beta: Angle beta
+       :param gamma: Angle gamma
+       :return: A unitcell object
+    '''
+    self._a = a
+    self._b = b
+    self._c = c
+    self._alpha = alpha
+    self._beta = beta
+    self._gamma = gamma
+
+   
+  @property
+  def a(self):
+    '''Returns dimension a of unit cell in Angstroms'''
+    return self._a
+
+  @property
+  def b(self):
+    '''Returns dimension b of unit cell in Angstroms'''
+    return self._b
+
+  @property
+  def c(self):
+    '''Returns dimension c of unit cell in Angstroms'''
+    return self._c
+
+  @property
+  def alpha(self):
+    '''Returns angle alpha of unit cell'''
+    return self._alpha
+
+  @property
+  def beta(self):
+    '''Returns angle beta of unit cell'''
+    return self._beta
+
+  @property
+  def gamma(self):
+    '''Returns angle gamma of unit cell'''
+    return self._gamma
+ 
+  def __str__(self):
+    '''Returns a pretty-printed object representation.'''
+    return ('\n'.join((
+      '  a         : {ir.a}',
+      '  b         : {ir.b}',
+      '  c         : {ir.c}',
+      '  alpha     : {ir.alpha}',
+      '  beta      : {ir.beta}',
+      '  gamma     : {ir.gamma}',
+    ))).format(ir=self)
```

Comment:
You've copied this literally. Since the variable ```ir``` here does not hold an ```IntegrationResult```, but a ```UnitCell``` I think ```uc``` would have probably been more appropriate.